app/services/steps_processor.rb

```diff
@@ -66,7 +66,7 @@ def load_steps(params)
 
   def load_step(step_klass, params)
     step_name = step_klass.name.underscore
-    step_params = params.fetch(step_name, {})
+    step_params = params.to_unsafe_h.fetch(step_name, {})
```

Comment:
Started to change this then got into a rabbit hole with the way Rails5 handles strong parameters in conjunction with our implementation of non-persistant models. I'm going to leave this as it is here and raise a ticket to tackle this later.